torch/_inductor/config.py

```diff
@@ -4,6 +4,8 @@
 
 import torch
 
+SUPPORTED_GPUS = ["cuda"]
+
```

Comment:
Should it be part of Triton section?

Reply:
Regarding the variable name, please follow the style of other variables of config.py.